logback/src/main/java/com/newrelic/logging/logback/NewRelicEncoder.java

```diff
@@ -28,7 +30,9 @@
  * @see <a href="https://logback.qos.ch/manual/encoders.html#interface">Logback Encoders</a>
  */
 public class NewRelicEncoder extends EncoderBase<ILoggingEvent> {
-    private NewRelicJsonLayout layout = new NewRelicJsonLayout();
+    private NewRelicJsonLayout layout;
+
+    Integer maxStackSize = MAX_STACK_SIZE;
```

Comment:
How does this get changed?

Reply:
I've been using Groovy too long. I'll add a setter for everyone else.